### DISCUSSION TITLE
Petition to add an official short alternative form of nush for nushell

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [nushell/nushell#10508] [Polls]
If `powershell` can be shorten to `pwsh`, why can't `nushell` be shorten to `nush`? 

Just kidding, you could name it anything all you want. Do keep in mind we are just adding another name and not changing it. You can easily do that in Linux today:
```
# ln -s /usr/bin/nushell /usr/bin/nush
```
However, on a serious thought, I still raised this poll so that we can have consistency.

**Comment by ghost** [comment]
I am confused. Which distribution or way of installing do you use that installs the nushell binary as `nushell` and not as `nu`?

**Reply by stevefan1999-personal** [reply]
Because `nu` is an interpreted configuration-based language, and `nushell` is a shell developed and integrated tightly with the `nu` language. They are equivalent in the end-user sense, as most people out here would just want a shell, but if you consider it logically, it's not entirely the same per-se. 

Therefore, on the contrary, I rarely use `nu` as it is rather more confusing to me, and I prefer the full name `nushell`, but I still think this full name is a bit too long.

**Reply by ghost** [reply]
Are you referring to `nuon` perhaps?

**Reply by stevefan1999-personal** [reply]
No, `nuon` is more of `json`. `nushell` is a shell implementing `nu` language as an interactive, interpreted shell. This is the same problem as "would you call a Lisp-based shell Lisp"?

**Reply by amtoine** [reply]
to the best of my knowledge, we do not ship any binary / executable called `nushell`, we only have `nu` :thinking:

**Reply by stevefan1999-personal** [reply]
Well don't you think `nu` is a bit misleading considering the points above? This is the difference like C and GCC...

**Reply by amtoine** [reply]
i see what you mean, yeah :+1: 

we'd have
- Nu as the language, just as we have now
- `.nu` as the extension
- `nushell` / `nush` as the binary / executable

personally, i think i do not feel this too much because `nu` runs automatically for me on startup for everything and i make scripts executable, but i can see the confusiong between the language and the tools used here :blush:
